test/manifold_test.cpp

```diff
@@ -784,3 +784,56 @@ TEST(Manifold, EmptyHull) {
       {0, 0, 0}, {1, 0, 0}, {0, 1, 0}, {1, 1, 0}};
   EXPECT_TRUE(Manifold::Hull(coplanar).IsEmpty());
 }
+
+TEST(Manifold, ConvexConvexMinkowski) {
+  bool oldDeterministic = ManifoldParams().deterministic;
+  ManifoldParams().deterministic = true;
+  Manifold sphere = Manifold::Sphere(0.1, 20);
+  Manifold cube = Manifold::Cube({2.0, 2.0, 2.0});
+  Manifold sum = cube.MinkowskiSum(sphere);
+  EXPECT_FLOAT_EQ(sum.GetProperties().volume, 10.589364f);
+  Manifold difference =
+      Manifold::Cube({2.0, 2.0, 2.0}).MinkowskiDifference(sphere);
+  EXPECT_FLOAT_EQ(difference.GetProperties().volume, 5.8319983f);
+  ManifoldParams().deterministic = oldDeterministic;
+}
+
+TEST(Manifold, NonConvexConvexMinkowski) {
+  bool oldDeterministic = ManifoldParams().deterministic;
+  ManifoldParams().deterministic = true;
+  Manifold sphere = Manifold::Sphere(1.2, 20);
+  Manifold cube = Manifold::Cube({2.0, 2.0, 2.0}, true);
+  Manifold nonConvex = cube - sphere;
+  Manifold sum = nonConvex.MinkowskiSum(Manifold::Sphere(0.1, 20));
+  EXPECT_FLOAT_EQ(sum.GetProperties().volume, 4.8406339f);
+  Manifold difference =
+      nonConvex.MinkowskiDifference(Manifold::Sphere(0.1, 20));
+  EXPECT_FLOAT_EQ(difference.GetProperties().volume, 0.2029168f);
+  ManifoldParams().deterministic = oldDeterministic;
+}
+
+/*
+TEST(Manifold, NonConvexNonConvexMinkowski) {
```

Comment:
Instead of commenting it out, just write DISABLED_NonConvexNonConvexMinkowski - that'll keep it compiling and let us know we have something to fix whenever we run the tests.